src/adapter/index.ts

```
@@ -1,4 +1,4 @@
-import type { EmojiGroup, EmojiSequence, EmojiVariation } from "../types";
+import type { Emoji, EmojiData, EmojiGroup, EmojiMetadata, EmojiSequence, EmojiShortcode, EmojiVariation, ShortcodeProvider } from "../types";
```

ESLint

[error] 'EmojiData' is defined but never used.

(unused-imports/no-unused-imports)